frontend/apps/case-entity/src/components/variants/occurrence/table/cnv-overlapping-gene-table-settings.tsx

```diff
@@ -0,0 +1,191 @@
+import { createColumnHelper } from '@tanstack/react-table';
+import { TFunction } from 'i18next';
+
+import { CNVGeneOverlap } from '@/api/api';
+import AnchorLinkCell from '@/components/base/data-table/cells/anchor-link-cell';
+import DocumentSizeCell from '@/components/base/data-table/cells/document-size-cell';
+import { createColumnSettings, TableColumnDef } from '@/components/base/data-table/data-table';
+import TooltipHeader from '@/components/base/data-table/headers/table-tooltip-header';
+
+import OverlapTypeGeneCell from './cells/overlap-type-gene-cell';
+
+const columnHelper = createColumnHelper<CNVGeneOverlap>();
+
+function getCNVOverlappingGenesColumns(t: TFunction<string, undefined>) {
+  return [
+    columnHelper.group({
+      id: 'unamed',
+      columns: [
+        // Genes
+        columnHelper.accessor(row => row.symbol, {
+          id: 'symbol',
+          cell: info => (
+            <AnchorLinkCell
+              href={`https://www.omim.org/search?index=entry&start=1&limit=10&sort=score+desc%2C+prefix_sort+desc&search=${info.getValue()}`}
+              external
+              target="_blank"
+            >
+              {info.getValue()}
+            </AnchorLinkCell>
+          ),
+          header: () => (
+            <TooltipHeader tooltip={t('variant.headers.symbol_tooltip')}>{t('variant.headers.symbol')}</TooltipHeader>
+          ),
+          size: 124,
+          minSize: 40,
+          enableSorting: false,
+        }),
+        // Cytoband
+        columnHelper.accessor(row => row.cytoband, {
+          id: 'cytoband',
+          header: t('variant.headers.cytoband'),
+          size: 124,
+          minSize: 40,
+          enableSorting: false,
+        }),
+        // ClinGen
+        columnHelper.accessor(row => row.symbol, {
+          id: 'clingen',
+          cell: info => (
+            <AnchorLinkCell
+              href={`https://search.clinicalgenome.org/kb/genes?search=${info.getValue()}`}
```

Comment:
I'm not sure but i think this link is often used in application. Maybe in the future, refactor to have a helper  function to build this link